### PR TITLE
screens: deprecate

### PR DESCRIPTION
**Pull request by unitof**
Has been Mac App Store only since launch of Screens 5

Copied language from agenda cask

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [ ] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/s/screens.rb [hunk 1]
```diff
@@ -7,10 +7,7 @@
   desc "Remote access software focusing on usability"
   homepage "https://edovia.com/screens-mac/"
 
-  livecheck do
-    url "https://updates.edovia.com/com.edovia.screens#{version.major}.mac/appcast.xml"
-    strategy :sparkle
-  end
+  deprecate! date: "2024-07-09", because: :discontinued
```
Comment (khipp) [inline Casks/s/screens.rb:10]:
```suggestion
  deprecate! date: "2024-07-09", because: :moved_to_mas
```

**Review by khipp — COMMENTED** [review]
A symbol is available for casks moved to the Mac App Store.

File: Casks/s/screens.rb (hunk 1 above)
Comment (unitof) [reply Casks/s/screens.rb:10]: Oh, nice! Do we still keep the `caveat`, or remove?

Comment (khipp) [reply Casks/s/screens.rb]: A maintainer would have to chime in here, but it seems like casks with a similar `caveat` predate the symbol. Using both is likely redundant.

Comment (p-linnane) [reply Casks/s/screens.rb]: Symbol is fine. The caveat isn't necessary. Thank you both!

**Review by p-linnane — APPROVED** [review]

**Pull request merged**